tests/functional/provisioning/test_dev_path_mapping_file.py

```diff
@@ -792,3 +793,93 @@ def test_dev_path_mapping_heketi_node_delete(self):
             use_percent, use_percent_after,
             "Failed to execute IO's in the app pod {} after respin".format(
                 pod_name))
+
+    @pytest.mark.tier2
+    @podcmd.GlustoPod()
+    def test_dev_path_volume_expansion(self):
+        """Validate dev path mapping for file volumes expansion"""
+
+        pvc_size, pvc_amount, expand_pvc_size = 1, 2, 2
+        vol_names, pvc_names = [], []
+
+        # Create PVC's
+        sc_name = self.create_storage_class(allow_volume_expansion=True)
+
+        # openshift_ops.wait_for_pvcs_be_bound(self.node, pvc_names)
+        pvc_names = self.create_and_wait_for_pvcs(
+            pvc_size=pvc_size, pvc_amount= pvc_amount, sc_name=sc_name)
+        dc_name = self.create_dcs_with_pvc(pvc_names)
+        self.validate_file_volumes_count(
+            self.h_node, self.h_server, self.node_ip)
+
+        for pvc_name in pvc_names:
+            pv_name = openshift_ops.get_pv_name_from_pvc(self.node, pvc_name)
+            volume_name = openshift_ops.get_vol_names_from_pv(
+                self.node, pv_name)
+            vol_names.append(volume_name)
+
+        # Get pvs info
+        pvs_info_before = openshift_storage_libs.get_pvs_info(
+            self.node, self.node_ip, self.devices_list, raise_on_error=False)
+        self.detach_and_attach_vmdk(
+            self.vm_name, self.node_hostname, self.devices_list)
+        pvs_info_after = openshift_storage_libs.get_pvs_info(
+            self.node, self.node_ip, self.devices_list, raise_on_error=False)
+
+        # Compare pvs info before and after
+        for (path, uuid, vg_name), (_path, _uuid, _vg_name) in zip(
+                pvs_info_before[:-1], pvs_info_after[1:]):
+            self.assertEqual(
+                uuid, _uuid, "pv_uuid check failed. Expected:{},"
+                "Actual: {}".format(uuid, _uuid))
+            self.assertEqual(
+                vg_name, _vg_name, "vg_name check failed. Expected:"
+                "{}, Actual:{}".format(vg_name, _vg_name))
+
+        for pvc in pvc_names:
+            openshift_ops.resize_pvc(self.node, pvc, expand_pvc_size)
+            openshift_ops.verify_pvc_size(self.node, pvc, expand_pvc_size)
+
+        for vol_id in vol_names:
+            vol_info = heketi_ops.heketi_volume_info(
+                self.h_node, self.h_server, vol_id['heketi_vol'], json=True)
+            self.assertEqual(
+                vol_info["size"], expand_pvc_size, "Failed to validate volume"
+                " size. Expected {}, Actual{}".format(
+                    expand_pvc_size, vol_info["size"]))
+
+        for vol_name in vol_names:
```

Comment:
Remove this loop, you can use the above loop

Reply:
done